account_move_base_import/models/account_journal.py

```diff
@@ -0,0 +1,287 @@
+
+    receivable_account_id = fields.Many2one(
+        comodel_name='account.account',
+        string='Receivable/Payable Account',
+        help="Choose a receivable/payable account to use as the default "
+        "debit/credit account.")
+
+    used_for_completion = fields.Boolean(
+        string="Journal used for completion")
+
+    rule_ids = fields.Many2many(
+        comodel_name='account.move.completion.rule',
+        string='Auto-completion rules',
+        relation='account_journal_completion_rule_rel')
+
+    launch_import_completion = fields.Boolean(
+        string="Launch completion after import",
+        help="Tic that box to automatically launch the completion "
+        "on each imported file using this profile.")
+
+    def _get_rules(self):
+        # We need to respect the sequence order
+        return sorted(self.rule_ids, key=attrgetter('sequence'))
+
+    def _find_values_from_rules(self, calls, line):
+        """This method will execute all related rules, in their sequence order,
+        to retrieve all the values returned by the first rules that will match.
+        :param calls: list of lookup function name available in rules
+        :param dict line: read of the concerned account.bank.statement.line
+        :return:
+            A dict of value that can be passed directly to the write method of
+            the statement line or {}
+           {'partner_id': value,
+            'account_id: value,
+            ...}
+        """
+        if not calls:
+            calls = self._get_rules()
+        rule_obj = self.env['account.move.completion.rule']
+        for call in calls:
+            method_to_call = getattr(rule_obj, call.function_to_call)
+            result = method_to_call(line)
+            if result:
+                result['already_completed'] = True
+                return result
+        return None
+
+    @api.multi
+    def _write_extra_move_lines(self, parser, move):
+        """Insert extra lines after the main statement lines.
+
+        After the main statement lines have been created, you can override this
+        method to create extra statement lines.
+
+            :param:    browse_record of the current parser
+            :param:    result_row_list: [{'key':value}]
+            :param:    profile: browserecord of account.statement.profile
+            :param:    statement_id: int/long of the current importing
+              statement ID
+            :param:    context: global context
+        """
+        move_line_obj = self.env['account.move.line']
+        global_commission_amount = 0
+        total_amount = 0
+        for row in parser.result_row_list:
+            global_commission_amount += float(
+                row.get('commission_amount', '0.0'))
+            total_amount += float(
+                row.get('amount', '0.0'))
+        total_amount += global_commission_amount
+        partner_id = self.partner_id.id
```

Comment:
Actually, for this part, it's already the wanted goal ; the method is "_write_extra_move_lines", which could not even be commission lines, or the counterpart ; you could specify what to add to the file as you wanted (for example, you could have the file with both the commission line and the counterpart in the file itself) for each type of import.

So I'm for keeping the current method, as it allows for more flexibility.